prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -90,6 +90,8 @@ public async Task<int> UpdateSiteAsync(int siteId, SiteUpdateModel updatedSite)
         {
             var currentSite = await GetSiteAsync(siteId);
 
+            currentSite.ActiveBeforeRegistration = updatedSite.ActiveBeforeRegistration;
+
             _context.Entry(currentSite).CurrentValues.SetValues(updatedSite);
```

Comment:
this line assigns all "simple" values from the updated site to the existing site, like `string`s, `int`s, `date`s, and  `bool`s. Basically, it assigns any values that are on the `Site` table bot not child objects. So line 93 is not needed,